godtools/App/Flows/Lesson/LessonFlow.swift

```diff
@@ -158,12 +164,46 @@ class LessonFlow: ToolNavigationFlow, Flow {
         navigationController.pushViewController(lessonView, animated: animated)
         
         configureNavigationBar(shouldAnimateNavigationBarHiddenState: true)
+        
+        showSwipeTutorialIfNeeded()
+    }
+    
+    private func showSwipeTutorialIfNeeded() {
+        
+        appDiContainer.feature.lessonSwipeTutorial.domainlayer.getShouldShowLessonSwipeTutorialUseCase().shouldShowLessonSwipeTutorialPublisher()
+            .receive(on: DispatchQueue.main)
+            .first()
+            .sink { [weak self] shouldShowSwipeTutorial in
```

Comment:
Combine also has a delay to delay the sink which can be handy.  However, I'm good with leaving the block based timer here too. 

https://developer.apple.com/documentation/combine/just/delay(for:tolerance:scheduler:options:)

```.delay(for: .seconds(0.8), scheduler: DispatchQueue.main)```

Reply:
Ohh that's cool!  Good to know for next time!